### DISCUSSION TITLE
Decide a naming convention for the container > content pattern

### DISCUSSION THREAD

**Discussion by delapuente** [Qiskit/qiskit.org#2379] [General]
The `container > content` layout pattern is used across the web in repeated occasions. Some times we use `.component > .component__container`, others `.component-container > .component` or even `.component > .component__layout`.

Let's come up with a unique and BEM-compliant naming convention and use it consistently across the web.

**Comment by delapuente** [comment]
One of the most extended patterns I can see through the codebase is `.component > .component__container`. See https://github.com/Qiskit/qiskit.org/search?q=__container

**Comment by korgan00** [comment]
Yes, I think that is the best.
Additionally, if something needs a double-wrap, I like to call the second one `.component__content` because the first one use to contain decorators or control buttons, and the content usually could be substituted for a `<slot />`.
